app/javascript/components/pdf-exporter/component.spec.js

```diff
@@ -76,20 +74,19 @@ describe("<PdfExporter />", () => {
   };
 
   it("renders PdfExporter", () => {
-    const { component } = setupMockFormComponent(PdfExporter, { props });
-
-    expect(component.find(PdfExporter)).to.have.lengthOf(1);
+    setupMockFormComponent(PdfExporter, { props });
+    expect(screen.getAllByText(/exports.printed/i)).toHaveLength(2);
   });
 
   it("renders Logos", () => {
-    const { component } = setupMockFormComponent(PdfExporter, { props });
+    setupMockFormComponent(PdfExporter, { props });
 
-    expect(component.find(Logos)).to.have.lengthOf(2);
+    expect(screen.getAllByText((content, element) => element.tagName.toLowerCase() === "svg")).toHaveLength(1);
```

Comment:
`content` variable is not being used, add a prefix or use regular expresion

Reply:
Done